crates/bevy_window/src/lib.rs

```diff
@@ -118,6 +118,18 @@ impl Plugin for WindowPlugin {
         if settings.close_when_requested {
             app.add_system(close_when_requested);
         }
+
+        #[cfg(debug_assertions)]
+        app.add_system(|wd: Option<Res<WindowDescriptor>>| {
+            if let Some(wd) = wd {
+                if wd.is_added() {
+                    warn!(
+                        "The WindowDescriptor resource must be inserted before the WindowPlugin is added. \
+                        Make sure to insert WindowDescriptor before adding the DefaultPlugins.",
```

Comment:
```suggestion
                        "The WindowDescriptor resource must be inserted before the WindowPlugin is added. \
                        Make sure to insert WindowDescriptor before adding the DefaultPlugins or the WindowPlugin.",
```
In case someone isn't using `DefaultPlugins`